### PR TITLE
[WIP] SIG Security Annual Report 2022

### LINKED ISSUE #7115: 2022 Annual Report: SIG Security
Chairs: @IanColdwater @tabbysable 
Liaison: @cpanato

Actions for the chair/organizer of the community group:
- [ ] Consult your community group to complete draft of report
- [ ] If needed, consult with Steering Committee liaison on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your Steering Committee liaison for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, Steering Committee will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

A template has already been generated for your group, and is available [here](https://git.k8s.io/community/sig-security/annual-report-2022.md).

Key dates:
- Initial PR to communtiy repo should be opened by March 24th, 2023
- PR should be reviewed and merged by April 7th, 2023

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

Your group's report template is available here: https://git.k8s.io/community/sig-security/annual-report-2022.md

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liaison directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/sig security
/committee steering
/area annual-reports

### PR DESCRIPTION
**Pull request by tabbysable**
**Which issue(s) this PR fixes**:

Fixes #7115

### CODE REVIEW

File: sig-security/annual-report-2022.md [hunk 1]
```diff
@@ -2,101 +2,100 @@
 
 ## Current initiatives
 
-1. What work did the SIG do this year that should be highlighted?
+**1. What work did the SIG do this year that should be highlighted?**
 
-   -
-   -
-   -
+- WIP
```
Comment (cpanato) [inline sig-security/annual-report-2022.md:7]: There are a few WIPs in the annual report, any update on those? thanks!

**Comment by palnabarun** [comment]
/uncc

(since Carlos is the liaison for SIG Security)

File: sig-security/annual-report-2022.md (hunk 1 above)
Comment (cpanato) [reply sig-security/annual-report-2022.md:7]: Gently ping back to close the pending items. thanks!

Comment (cpanato) [reply sig-security/annual-report-2022.md:7]: @tabbysable gently ping to check the status here, thanks

**Pull request closed without merging**